Trappist/src/Promact.Trappist.Repository/TestSettings/TestSettingsRepository.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Promact.Trappist.DomainModel.DbContext;
+using Promact.Trappist.DomainModel.Models.Test;
+using System.Collections.Generic;
+using System.Linq;
+
+
+namespace Promact.Trappist.Repository.TestSettings
+{
+    public class TestSettingsRepository : ITestSettingsRepository
+    {
+        private readonly TrappistDbContext _dbContext;
+        Test test = new Test();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the 'test' field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1450)